e2e/ChannelApeClient.spec.ts

```diff
@@ -8,20 +8,26 @@ import { LogLevel } from 'channelape-logger';
 
 describe('ChannelApe Client', () => {
 
+  beforeEach((done) => {
+    setTimeout(() => done(), 100);
```

Comment:
Please remove this it is not needed. The build is already really slow at this point.